sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc

```diff
@@ -161,15 +161,15 @@ a|
 intel_gpu_bdw
 ----
 |-
-|Broadwell Intel graphics architecture.
+|Intel(R) microarchitecture code name Broadwell Intel graphics architecture.
 
 a|
 [source]
 ----
 intel_gpu_skl
 ----
 |-
-|Skylake Intel graphics architecture.
+|Intel(R) microarchitecture code name Skylake Intel graphics architecture.
```

Comment:
Is there a reason not to add this same wording to all the other architecture names below?

Reply:
In short, yes, answered internally